### DISCUSSION TITLE
Best procedure for NAMD

### DISCUSSION THREAD

**Discussion by koroki** [nwchemgit/nwchem#658] [Q&A]
Following the original paper of NAMD in nwchem ( https://pubs.acs.org/doi/10.1021/acs.jctc.0c00295 ) and also recommended by different other works, the initial conditions are critical to start a NAMD simulation. 
Usually, an snapshot of a previous MD simulation correctly thermostated is used to initialized an NAMD, using positions and velocities. 

However, I could not find the proper way to do it in nwchem. I found this post in an old forum (I guess):
https://nwchemgit.github.io/Special_AWCforum/st/id1878/Initial_velocities_in_QMD.html
where the restart file, .qmdrst, can be used. After test and error (this part is not well explained on manual), I understand how to use restart files in the nwchem AIMD simulations. 
However, the .qmdrst file for NAMD is different and the qmdrst of "normal" MD simulations cannot be used. If the non NAMD simulation qmdrst is used on NAMD, nwchem fails for end of file error. Opening the qmdrst file in NAMD, it includes more information such as the number of states and a list of values that describes the density (with density?). 

So, at this situation. With is the best way to perform this simulations?

**Comment by koroki** [comment]
After some days of digging in the manual I noticed that one can also specify the velocity in the geometry section:
https://nwchemgit.github.io/Geometry.html

It is not easy to find since this information is not written in the text and only is visible as [vx vy vz].
I will try to modify this information in the manual because also the units of this velocity are not mentioned. 

This is not the easiest way because the velocity must be introduced in au, while the velocity is saved on A/fs on the xyz. But is works!

**Comment by ngovind** [comment]
Hi, We have not fully integrated our NAMD implementation described in the
JCTC paper into the released code yet. I plan to do that soon.
You are correct that the .qmdrst from a normal MD run cannot be used for
NAMD. NAMD restarts will look for additional information like the state
information, density matrix, and coupling between the states. This
information is currently written into the .qmdrst file in the released code
if you perform a NAMD run.

Can you send me your input file ?

Best,
-Niri

Niri Govind
PNNL
***@***.***

**Reply by koroki** [reply]
Hi Niri,
As I wrote before here, I think that the easy solution is using the geometry input section.

For now, the input is quite simple. I will send via email.

**Comment by ngovind** [comment]
Sounds good. You could run an initial normal MD to thermally equilibration
to get the initial velocities.
In parallel, you can also start a NAMD run for a few steps to generate a
.qmdrst file. Once you have a .qmdrst file, you can swap in the
equilibrated velocities and restart.

Best,
-Niri
